services/node/node-base.js

```diff
@@ -15,6 +15,13 @@ module.exports = class NodeVersionBase extends NPMBase {
 
   static get examples() {
     const type = this.type
+    const documentation = `
+  <p>
+    This badge is designed to covey to users the node version support status of a package.
```

Comment:
I think we can drop this first sentence as it's essentially covered by the second/class specific sentence
```suggestion
```